docs/tutorials.rst

```diff
@@ -19,10 +19,20 @@ STAC Metadata Visualization
 This tutorial gives an introduction to using Holeviews and hvplot to visualize
 STAC metadata and Item geometries on a map.
 
-CQL Filtering
+CQL2 Filtering
 ---------------------------
 
 - :tutorial:`GitHub version <cql2-filter.ipynb>`
 
 This tutorial gives an introduction to using CQL-JSON filtering in searches to
 search by arbitrary STAC Item properties.
+
+Calculating Coverage Percentage of the AOI by an Item
+-----------------------------------------------------
+
+- :tutorial:`GitHub version <aoi-coverage.ipynb>`
```

Comment:
```suggestion
- :tutorial:`GitHub version <aoi-coverage.ipynb>`
- :ref:`Docs version </tutorials/aoi-coverage.ipynb>`
```

I tried this locally and I _think_ it works, but can you check?

Reply:
are all of them supposed to have that?

Reply:
I added it to all of them, and fixed the ones that were broken. even the one in current docs was broken https://pystac-client.readthedocs.io/en/latest/tutorials/pystac-client-introduction.html -- see the errors